src/dashboards/Github projects-1667481831135.json

```diff
@@ -1,761 +0,0 @@
-{
-  "__inputs": [
```

Comment:
was this intentionally removed?

Reply:
Yes, I checked the dashboard and it is not very usable and other dashboard is much better
<img width="1911" alt="image" src="https://github.com/grafana/github-datasource/assets/30407135/25efa2a1-80b1-4185-9131-591b5439031f">